### PR TITLE
fixes everything

### PR DESCRIPTION
**Pull request by taboularasa**

### CODE REVIEW

**Comment by cltatman** [comment]
great change :+1:

**Comment by lexi-lambda** [comment]
:-1: this doesn't fix Chrome **and** it breaks on Safari, too

**Comment by taboularasa** [comment]
:+1:

**Comment by cltatman** [comment]
yeah :+1:

**Comment by taboularasa** [comment]
:ok_hand:

**Comment by cltatman** [comment]
:shipit:

**Comment by lexi-lambda** [comment]
:no_good:

File: README.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-HI
-HI ➕1️⃣
-
-Eddie is 🆒:
```
Comment (lexi-lambda) [inline README.md]: @taboularasa what was the rationale behind removing this line?

Comment (taboularasa) [inline README.md]: :tada:

Comment (cltatman) [inline README.md]: @lexi-lambda favoritism is ugly

**Comment by taboularasa** [comment]
what?

**Comment by cltatman** [comment]
giphy fail

**Comment by jadengore** [comment]
Giphy works on Github? :astonished:

File: README.md (hunk 1 above)
Comment (jadengore) [inline README.md]: Wow, sometimes you just want to feel :heart_eyes:

**Comment by jadengore** [comment]
@pgdaniel Can you reopen this?

**Comment by pgdaniel** [comment]
@jadengore won't let me :-/

**Pull request closed without merging**